.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
It adds stuff to the gitignore that isn't necessary. 

project gitignores are meant to house things that should be ignored for that specific project